fastlane/lib/fastlane/actions/adb_devices.rb

```diff
@@ -68,6 +68,15 @@ def self.authors
       def self.is_supported?(platform)
         platform == :android
       end
+
+      def self.is_compatible?(operating_system)
+        compatibility = {
+          "MacOS"   => true,
+          "Linux"   => false,
+          "Windows" => true
+        }
+        super(operating_system, compatibility)
```

Comment:
Is it possible to do this without calling `super`? I don't think we use super anywhere else so not sure if we want to try and make things consistent when people try to implement them? But I do also like this approach too 🤔 

Is it possible to also just do something like...
```rb
def self.is_compatible?(operating_system)
  [:macos, :windows].include?(operating_system)
end
```

Reply:
> Is it possible to do this without calling super?

I don't think so as I wanted it to have a "default compatibility" that is present for all actions by default - which would currently be all of them. 

(The `is_supported?` which we already have and is a bit similar doesn't need that as it was required from the beginning and calling an action without it actually stops execution with an error message)

I actually like this better as this way each action doesn't "replicate" the logic to check - but of course this is only important here as the checking logic is a bit more complicated with the defaults.

> Is it possible to also just do something like...
> `[:macos, :windows].include?(operating_system)`

I think using just an array only works for yes/no situations. But be have yes/no/unknown to express that it 1) works, 2) doesn't work or 3) hasn't been explicitly set/tested.